Solutions/Cybersixgill-Actionable-Alerts/Playbooks/CybersixgillAlertStatusUpdate/readme.md

```diff
@@ -0,0 +1,27 @@
+# CybersixgillAlertStatusUpdate
+author: Loginsoft
+
+This playbook will update status of Actionable alerts in Cybersixgill Portal. When incident is updated in Microsoft Sentinel, playbook will run and update status Actionable alerts from Cybersixgill Portal 
+
+# Prerequisites
+We will need the following data to do one time setup
+
+1. Cybersixgill Client ID (client_id)
+2. Cybersixgill Client Secret (client_secret)
+

```

Comment:
Please provide more info. On how customer can fetch the above-mentioned details from  Cybersixgill  UI /portal